### PR TITLE
shiarium devserver chain id added

### PR DESCRIPTION
**Pull request by navdeepgarg1**
shiarium devserver chain id added

### CODE REVIEW

**Comment by realdealshaman** [comment]
Hey, thanks for the PR. Request to add a new network should be added here first: https://github.com/ethereum-lists/chains Once it is accepted there, it will appear on our site also

**Comment by realdealshaman** [comment]
Hey @navdeepgarg1 , did you submit a PR to the repo in the url I provided?

**Comment by Martins457** [comment]
Pls, how do I connect my wallet

File: constants/extraRpcs.json [hunk 1]
```diff
@@ -1378,5 +1378,10 @@
             "https://evm.cryptocurrencydevs.org",
             "https://rpc.crystaleum.org"
         ]
+    },
```
Comment (AZIMUT98155) [inline constants/extraRpcs.json:1381]: shibaswaparmy:feature/shibarium-chain

File: constants/extraRpcs.json [hunk 2]
```diff
@@ -1378,5 +1378,10 @@
             "https://evm.cryptocurrencydevs.org",
             "https://rpc.crystaleum.org"
         ]
+    },
+    "7352":{
+        "rpc":[
```
Comment (AZIMUT98155) [inline constants/extraRpcs.json:1383]: shibaswaparmy:feature/shibarium-chain

**Review by AZIMUT98155 — APPROVED** [review]

**Comment by realdealshaman** [comment]
Have you submitted a PR here: https://github.com/ethereum-lists/chains ?

**Comment by realdealshaman** [comment]
Closing this PR since a PR is not needed to add a new network. The new network will appear automatically once your PR is submitted and merged in the ethereum-lists repo. Please use this repo only if you would like to add new rpc endpoints to chains that have already been merged

**Pull request closed without merging**